07_Agents/00_LLM_Claude_Agent_Tools.ipynb

```diff
@@ -59,18 +71,30 @@
    },
    "outputs": [],
    "source": [
-    "import boto3\n",
     "import json\n",
     "import os\n",
     "import sys\n",
+    "import boto3\n",
     "\n",
     "module_path = \"..\"\n",
     "sys.path.append(os.path.abspath(module_path))\n",
     "from utils import bedrock, print_ww\n",
     "\n",
-    "os.environ['AWS_DEFAULT_REGION'] = 'us-east-1'\n",
-    "os.environ['SERPAPI_API_KEY'] = '<enter serp api>'\n",
-    "boto3_bedrock = bedrock.get_bedrock_client(os.environ.get('BEDROCK_ASSUME_ROLE', None))"
+    "\n",
+    "# ---- ⚠️ Un-comment and edit the below lines as needed for your AWS setup ⚠️ ----\n",
+    "\n",
+    "# os.environ[\"AWS_DEFAULT_REGION\"] = \"<REGION_NAME>\"  # E.g. \"us-east-1\"\n",
+    "# os.environ[\"AWS_PROFILE\"] = \"<YOUR_PROFILE>\"\n",
+    "# os.environ[\"BEDROCK_ASSUME_ROLE\"] = \"<YOUR_ROLE_ARN>\"  # E.g. \"arn:aws:...\"\n",
+    "# os.environ[\"BEDROCK_ENDPOINT_URL\"] = \"<YOUR_ENDPOINT_URL>\"  # E.g. \"https://...\"\n",
```

Comment:
With Bedrock going GA, we don't need bedrock endpoint url, lets verify and remove.

Reply:
Updated